__fixtures__/verifiable-project/packages/app-one/package.json

```diff
@@ -6,9 +6,9 @@
   },
   "dependencies": {
     "package-one": "workspace:*",
-    "package-two": "workspace:^",
```

Comment:
Did you need to change this? I think by having a vareity of semver syntax, it helps increase confidence in our semver checking logic

Reply:
This is another test fixture, specifically made to `verifyWorkspaceDependencies`.
I think @steveukx just copy-pasted the original fixture, `clean-workspace-3`, that you created to test mismatched dependencies.
In the world pre-this PR, `verifyWorkspaceDependencies` errored straight away because one of the workspaces was outside `packages/`.
In this PR, we want to assert that `verifyWorkspaceDependencies` doesn't error if one of the workspaces is outside `packages/`. The problem is that now the check proceeds, finds mismatched dependencies and errors anyway. That's why I need all dependencies being matched in this specific case.

Reply:
Understood 👍